content/zh/docs/reference/config/security/conditions/index.md

```diff
@@ -6,7 +6,7 @@ aliases:
     - /zh/docs/reference/config/security/conditions/
 ---
 
-此页面描述了可以用作[授权策略](/zh/docs/reference/config/security/authorization-policy/) `when` 字段中所支持的键和值的格式。
+此页面描述了可以用作[授权策略](/zh/docs/reference/config/security/authorization-policy/#Condition) `when` 字段中所支持的键和值的格式。
```

Comment:
Remove this file from this PR. Leave the zh version unchanged. The Chinese translators can change it later if they wish. In general, unless we're just fixing broken links, we should avoid updating translated docs in the same PR as the English change.

Reply:
Alright! Thanks for the tip :+1: